### PR TITLE
[Refactor] Mutez to Tez

### PR DESCRIPTION
**Pull request by JulesGuesnon**
# Description

This PR remove all mutez references and replace everything by Tez so it's easier for users.

There's one important question that needs to be answer tho: should we write `xtz` or `tez` everywhere?

There's also some small UI fixes in it

### CODE REVIEW

**Comment by rueshyna** [comment]
> There's one important question that needs to be answer tho: should we write xtz or tez everywhere?

There are 3 symbol, actually.
- Tzkt uses `ꜩ` : https://tzkt.io/tz3gLTu4Yxj8tPAcriQVUdxv6BY9QyvzU1az/operations/
- Temple wallet uses `TEZ`
- TzStats uses `XTZ`

I think prefer `TEZ`. with `TEZ`, if want to present `MUTEZ` one day, it seems to be easier. Maybe it won't be case. 
The second preference of mine is `ꜩ`
The button line is we choice only one and always use it.

File: utils/tez.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import BN from "bignumber.js";
+
+const MUTEZ_TEZ_RATIO = Math.pow(10, 6);
+
+export const mutezToTez = (mutez: number): number =>
```
Comment (rueshyna) [inline utils/tez.ts:5]: Can you add some unit test for these two functions?
and make CI run test when build?

File: versioned/interface.ts [hunk 2]
```diff
@@ -197,7 +197,7 @@ abstract class Versioned {
         fields: [
           {
             field: "amount",
-            label: "Amount in Mutez",
```
Comment (rueshyna) [inline versioned/interface.ts:200]: I think adding unit is need so maybe like `Amount(Tez)`?

**Review by rueshyna — COMMENTED** [review]
The others looks good to me

File: versioned/interface.ts (hunk 2 above)
Comment (rueshyna) [reply versioned/interface.ts:200]: so as the others

**Comment by JulesGuesnon** [comment]
@rueshyna I added the test inside the dockerfile, so everything will stop if tests fails

**Review by rueshyna — APPROVED** [review]
Please change xtz to Tez before merge
```
> ag xtz 
components/topUpForm.tsx
100:                  Failed to transfer xtz
108:            Amount of xtz to transfer

components/Sidebar.tsx
71:        <p className="text-lg">{formattedBalance.toString()}xtz</p>
```

**Pull request merged**